packages/pn-pa-webapp/src/pages/NotificationDetail.page.tsx

```diff
@@ -79,6 +87,8 @@ const NotificationDetail = () => {
 
   const hasNotificationSentApiError = hasApiErrors(NOTIFICATION_ACTIONS.GET_SENT_NOTIFICATION);
 
+  const [timeoutMessage, setTimeoutMessage] = useState(0);
```

Comment:
Instead of the useState, i would create the const timeuotMessage = legalFactDownloadRetryAfter * 1000. This const will be updated each time legalFactDownloadRetryAfter changes